src/test/java/racingGame/Business/CarFactoryTest.java

```diff
@@ -0,0 +1,22 @@
+package racingGame.Business;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import racingGame.model.Rules;
+
+class CarFactoryTest {
+    
+    private CarFactory carFactory;
+    private Rules rules;
+    
+    // 자동차 입력받은만큼 생성했는지?
+    @ParameterizedTest
+    @ValueSource(ints = {3, 4, 5, 6})
+    void carMakeTest(int number) {
+        rules = new Rules(number,1);
+        carFactory = new CarFactory(rules);
```

Comment:
```suggestion
        List<Car> cars = CarFactory.createCars(number);
```
이와 같이 단순하게 구현할 수 있지 않을까?
Rules까지 참여하다보니 복잡도가 높아 보임